gnd/src/main/java/com/google/android/gnd/repository/local/Edit.java

```diff
@@ -60,11 +60,19 @@ public static int toInt(@Nullable Edit.Type value) {
   @NonNull
   public abstract Type getType();
 
+  /**
+   * For edits of type UPDATE, returns a JSON object with the original value of modified attributes
```

Comment:
Nit: maybe use code font for UPDATE (since it's capitalized to represent the code symbol).

Reply:
Added link instead